ansible/install_romm_cifs.yml

```diff
@@ -0,0 +1,44 @@
+---
+- hosts: localhost
+  gather_facts: false
+
+  vars:
+    my_name: "RomM CIFS"
+    my_file: "install_romm_cifs"
+    module_name: "romm_cifs"
+    system_key: "romm"
+
+    top_level_paths:
+      - { name: "roms",   enabled: yes,  generic: "roms",   systems: yes }
+
+    templates:
+      - { name: "retronas-romm-dirs.service", dest: "/etc/systemd/system" }
+      - { name: "retronas-romm-dirs.timer", dest: "/etc/systemd/system" }
+
+    systemd_units:
+      - { name: "retronas-romm-dirs", type: 'service', state: "stopped", enabled: "no", restart: "no", instance: "no" }
+      - { name: "retronas-romm-dirs", type: 'timer',   state: "stopped", enabled: "no", restart: "no", instance: "no" }
+
+  tasks:
+
+    - name: "{{ my_name }} - Load RetroNAS config"
+      ansible.builtin.include_vars: retronas_vars.yml
+
+    - ansible.builtin.import_role:
+        name: retronas.role.romdir
+
+    - ansible.builtin.import_role:
+        name: retronas.role.samba
+
+    - ansible.builtin.import_role:
+        name: retronas.role.samba.system.nolink
```

Comment:
I debated calling this system.romm but figured if another integration needs a flat list this could work too?

Reply:
see comment on role